index.md

```diff
@@ -1,7 +1,83 @@
-# Welcome to `nmstate`
+# We are nmstate!
+A descriptive network manager API for hosts.
 
-Hello hello hello.
+## What is it?
+A library with an accompanying cli tool that manages host networking settings
+in a descriptive manner. The networking state is described by a pre-defined
+schema, where reporting (current state) and changes (desired state) are both
+confirming to it.
+
+Aimed to satisfy enterprise needs to manage host networking through a
```

Comment:
Pleases use complete English sentence, including a subject (`nmstate` in this case)